faker/providers/person/en_IN/__init__.py

```diff
@@ -2,225 +2,554 @@
 
 
 class Provider(PersonProvider):
-    formats = ("{{first_name}} {{last_name}}",)
 
-    # First names are from
-    #   https://www.babycenter.in/a25010193/modern-indian-baby-names
```

Comment:
We need sources for the data. Please refer to our [coding style](https://github.com/joke2k/faker/blob/master/docs/coding_style.rst#data-sets).